README.md

```diff
@@ -0,0 +1,66 @@
+<img src="/doc-base/neo.png" align="right" height="70"/>
+
+# NEO Specification
+
+## document format
+The Community Specification Paper base document started as pure LaTeX (as a [Yellow Paper](https://github.com/NeoResearch/yellowpaper])), as usual in academic publications, however the syntax is not considered as friendly as Markdown. So, why not merge both? :)
+In order to facilitate the contributions to the document, we decided to adopt pandoc format, which allows hybrid LaTeX + Markdown syntax.
+
+### base structure
+The structure of the document consists of five files:
+- specification_paper.md : main document in Mardown + LaTeX (this file is automatically generated by `merge_sections.sh`)
```

Comment:
Can we change the name to `specification.md`?

Reply:
Done